tests/testthat/test-derive_params_growth_height.R

```diff
@@ -0,0 +1,41 @@
+# derive_params_growth_heightlength ----
+
+## Test 1: derive_params_growth_heightlength works ----
```

Comment:
Some outliers data to be added

Reply:
Additional testing will be handled in #50 #51